### DISCUSSION TITLE
Question: Exclude type definitions in completion items

### DISCUSSION THREAD

**Discussion by anonimitoraf** [ananthakumaran/tide#437] [Q&A] [answered]
Hi!

Thank you for this package. Is there any way to disable these type definitions?
I think they're redundant given that the types are also in the annotations (on the right)
![image](https://user-images.githubusercontent.com/15933322/153295365-45180a96-a1a7-4ec3-b15a-2935c93e1685.png)

**Comment by ananthakumaran** [comment, marked as answer]
Do you have `(setq tide-completion-detailed t)`? By default, this info is not shown unless the flag is enabled.

**Comment by anonimitoraf** [comment]
Sweet, thanks. That did the trick. I didn't know `t` was the default

**Reply by ananthakumaran** [reply]
https://github.com/ananthakumaran/tide/blob/master/tide.el#L157 Default is `nil`. But if you use some framework, it might modify the default
